conf/kubernetes.conf

```diff
@@ -1,6 +1,6 @@
-<match fluent.**>
+<label @FLUENT_LOG>
```

Comment:
@yyyogev I think we need to revert this or use this pattern instead: https://docs.fluentd.org/deployment/logging#capture-fluentd-logs